CUSTOMIZATION.md

```diff
@@ -80,3 +83,10 @@ The header may optionally define the following:
 
 *   `etherpad` is the URL for the Etherpad for your workshop.  If you are
     not using an Etherpad, you can delete this value.
```

Comment:
Eventbrite and etherpad is in opposite order in index.html than it is here, in CUSTOMIZATION.md, which might cause confusion. Also, it says you can delete etherpad - what happens if you delete eventbrite?